src/build.jl

```diff
@@ -215,32 +215,62 @@ function solve!(m::AbstractMPSGEModel; kwargs...)
 
 
     for (k,v) in kwargs
-
         JuMP.set_attribute(jm, string(k), v)
     end
 
+    # Unfix numeraire, if set
+    if !ismissing(m.numeraire) 
+        unfix(numeraire(m))
+    end
+
+    # Update consumer start values
+    for consumer∈consumers(m)
+        if termination_status(jm) == OPTIMIZE_NOT_CALLED
```

Comment:
If the model hasn't been optimized then we are unable to call the standard `value`. The fix is to use the starting values of each variable.